### DISCUSSION TITLE
Markdown Standards

### DISCUSSION THREAD

**Discussion by dskiles** [dskiles/eloransetting#22] [General]
@CSLewin 

We should decide on a markdown standard.

I think the two main contenders are [github-flavored markdown](https://github.github.com/gfm/), and [commonmark](https://commonmark.org/).

Since tables feature heavily, I think we need the former.

**Comment by CSLewin** [comment]
I do want lots of 3+ element tables (X+Y+Z) for varied outcomes, so that tracks. Github-flavored markdown.

**Reply by dskiles** [reply]
SO LET IT BE WRITTEN

SO LET IT BE DONE
